core/src/main/java/hudson/views/ListViewColumn.java

```diff
@@ -125,14 +126,14 @@ public boolean shownByDefault() {
         // OK, set up default list of columns:
         // create all instances
         ArrayList<ListViewColumn> r = new ArrayList<ListViewColumn>();
-
+        final JSONObject emptyJSON = new JSONObject();
         for (Descriptor<ListViewColumn> d : ListViewColumn.all())
             try {
                 if (d instanceof ListViewColumnDescriptor) {
                     ListViewColumnDescriptor ld = (ListViewColumnDescriptor) d;
                     if (!ld.shownByDefault())       continue;   // skip this
                 }
-                ListViewColumn lvc = d.newInstance(null, null);
+                ListViewColumn lvc = d.newInstance(null, emptyJSON);
```

Comment:
This one is potentially serious (violation of safe-defined API for plugins), but I do not see related issues